tests/test_api.py

```diff
@@ -31,3 +31,10 @@ def tracer(self):
 
     def check_baggage_values(self):
         return True
+
+    def is_parent(self, parent, span):
+        # use `Span` ids to check parenting
+        if parent is None:
+            return span.parent_id is None
```

Comment:
Covers the case where `is_parent(None, span)` returns `True` if `span` is a root span.